deploy/prometheus/prometheus-alert-rules.yaml

```diff
@@ -787,3 +787,89 @@ spec:
               ```
               kubectl logs -n {{ $labels.namespace }} pod/{{ $labels.pod }} -f | grep -i -E "tls|ssl|auth|cert|handshake"
               ```
+
+    - name: ClickHouseBackupRules
```

Comment:
latest version clickhouse-backup have different incompatible metrics schema, so, we need upgrade tests and alert rules when will switch to new version